src/tasks/build.js

```diff
@@ -65,26 +65,30 @@ module.exports = function() {
   // We need to replace the extension bridge from akamai with one that is bundled inside the build.
   fs.copySync(files.EXTENSION_BRIDGE_PATH, files.DIST_PATH);
 
+  var script = '<script>' +
+      'var basePath = window.parent.location.href.replace(/\\/[^\/]*$/, "");' +
+      'document.write(\'<script src="\' + basePath + \'\/extensionbridge.min.js"><\\/script><script src="\' + basePath + \'\/extensionbridge-child.js"><\\/script>\');' +
+    '</script>';
```

Comment:
Is this going to show a warning in the console because we're document.writing an script tag loading an external resource? Do we care?

Reply:
Do you have another idea on how to insert a dynamic HTML in a sync way?